wdl2cwl/tests/oop_cwl_files/bcftools_stats.cwl

```diff
@@ -0,0 +1,167 @@
+class: CommandLineTool
+id: Stats
+inputs:
+  - id: inputVcf
+    type: File
+  - id: inputVcfIndex
+    type: File
+  - id: outputPath
+    default: basename(inputVcf) + .stats
```

Comment:
This is not a valid CWL default, sorry. We can't use expressions here. This is an error in the original `bcftools_stats`

Reply:
Ok. I will revert to the former solution you suggested.